### DISCUSSION TITLE
Should maintainers of List-KR unblock ad servers in DNS filter

### DISCUSSION THREAD

**Discussion by piquark6046** [List-KR/community#13] [Polls]
Like anti-adblock and incorrect blocking issue, some website cannot be loaded correctly because they require some features included in blocked request by List-KR or other filter.

When DNS filtering is enabled, the issue is not resolved even if maintainers of List-KR unblock the request or redirect embed resource.

Therefore, I opened this poll. please vote.

**Comment by green1052** [comment]
when to end this vote? @piquark6046

**Reply by piquark6046** [reply]
On May 22.

**Reply by green1052** [reply]
@piquark6046

**Reply by piquark6046** [reply]
On May 22, I confirmed that `Create new DNS filter` wins by 1 vote.
I will create new project.

**Reply by green1052** [reply]
👍

**Reply by green1052** [reply]
> On May 22, I confirmed that `Create new DNS filter` wins by 1 vote. I will create new project.

@piquark6046 create new repo? like [energized host](https://github.com/EnergizedProtection/block)?

**Reply by piquark6046** [reply]
Nope. I will simply create new folder.

**Reply by green1052** [reply]
![다운로드](https://user-images.githubusercontent.com/34261355/173847099-ac9ceee9-6436-4b65-85a2-87a37689bd66.jpeg)
